test_runner.py

```diff
@@ -273,6 +273,39 @@ def build_test_list():
             "fsdp2_mem_tracker",
             ngpu=4,
         ),
+        OverrideDefinitions(
+            [
+                [
+                    "--training.fp8_linear",
+                ]
+            ],
+            "FSDP2 with bf16 all-gather",
```

Comment:
maybe noob question: does all-gather always happen in bf16, or is it determined by `param_dtype` of FSDP2

Reply:
good question! I should change this to "all-gather in original dtype"

when mixed_precision is turned off (no `param_dtype` configed), FSDP2 communicate in model's original dtype

when mixed_precision is turned on (`param_dtype`=`xxx`), FSDP2 communicates according to `param_dtype`